docs/integrations-lang-chain.mdx

```diff
@@ -0,0 +1,21 @@
+---
+id: genaiIntegrationsLangChain
+slug: /vector-gen-ai/integrations-lang-chain
+title: LangChain
+image:
+description: Learn how to integrate ((es)) with LangChain
+date: 2023-09-07
+tags: ['n']
+related: ['genaiIntegrationsOpenAI', 'genaiIntegrationsLlamaIndex', 'genaiIntegrationsHuggingFace']
+---
+
+[LangChain](https://docs.langchain.com/docs/) is a popular framework for working with AI, Vectors, and embeddings. Used to simplify building a variety of AI applications.
+
+((es)) can be used with LangChain in two ways:
+
+- Use the LangChain VectorStore to store and retrieve documents from ((es))
+- Use the LangChain self-query retriever, with the help of an LLM like OpenAI, to transform a user's query into a query + filter to retrieve relevant documents from ((es)).
+
+<DocCallOut>
```

Comment:
I suggest we copy and paste the blurb from the notebooks readme.md into here. Likely we will now redirect developers to this microsite and they will enter elasticsearch-labs through this url

Reply:
@joemcelroy Sorry did you mean to add that blurb specifically on this langchain page or on the integrations overview page (`integrations-overview.mdx`)?

Thinking the notebooks/readme.md blurb should probably live in `notebooks.mdx` at least?